packages/yoga/src/Divider/native/Divider.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import theme from '../../Theme/helpers/themeReader';
+
+const StyledDivider = styled.View`
+  ${props => {
+    const { borders, colors } = theme(props);
+
+    return css`
+      height: ${borders.small};
+      width: 100%;
```

Comment:
If we need to add margins to this component the `width: 100%` prop will move the component but it will stay with the same size, causing a strange behaviour, in the past, we needed to add to other components the prop `flex: 1` instead of putting the `width: 100%` to avoid this behaviour.
Is it the same case?

Reply:
Actually, the `width: 100%` has a size based on the father component, so, if we put a margin on father component, it will be with 100% of the size inside the margin, like the following image. (I put more height to divider just  to a better visualization)
https://reactnative.dev/docs/height-and-width#percentage-dimensions
![image](https://user-images.githubusercontent.com/94182676/143574235-bdce0920-5641-44a8-bfe1-6327d9ea26cc.png)